submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/ResubmissionApiTest.kt

```diff
@@ -245,36 +247,65 @@ class ResubmissionApiTest(
     }
 
     @Test
-    fun `5-5 super user suppresses submission from another user`() {
-        val version1 =
-            tsv {
-                line("Submission", "S-RSTST5")
-                line("Title", "Public Submission")
-                line("ReleaseDate", OffsetDateTime.now().toStringDate())
-                line()
-                line("Study")
-                line()
-            }.toString()
+    fun `5-5 super user suppresses submission from another user`() =
```

Comment:
unrelease

Reply:
I don't think we should change it in this case since the integration tests represent the business level operation and in that level it was decided the name for the operation to make a public submission private would be "suppress"